test/e2e/runner/load.go

```diff
@@ -95,32 +96,56 @@ func loadGenerate(ctx context.Context, chTx chan<- types.Tx, multiplier int, siz
 func loadProcess(ctx context.Context, testnet *e2e.Testnet, chTx <-chan types.Tx, chSuccess chan<- types.Tx) {
 	// Each worker gets its own client to each node, which allows for some
 	// concurrency while still bounding it.
-	clients := map[string]*rpchttp.HTTP{}
+
+	clients := make([]*rpchttp.HTTP, 0, len(testnet.Nodes))
+
+	for idx := range testnet.Nodes {
+		if testnet.Nodes[idx].Mode == e2e.ModeSeed {
+			continue
+		}
+		client, err := testnet.Nodes[idx].Client()
+		if err != nil {
+			continue
+		}
+
+		clients = append(clients, client)
+	}
+
+	if len(clients) == 0 {
+		panic("no clients to process load")
+	}
+
+	clientRing := ring.New(len(clients))
+	for idx := range clients {
+		clientRing.Value = clients[idx]
+		clientRing = clientRing.Next()
+	}
 
 	var err error
-	for tx := range chTx {
-		node := testnet.RandomNode()
 
-		client, ok := clients[node.Name]
-		if !ok {
-			client, err = node.Client()
-			if err != nil {
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case tx := <-chTx:
+			clientRing = clientRing.Next()
+			client := clientRing.Value.(*rpchttp.HTTP)
+
+			if _, err := client.Health(ctx); err != nil {
 				continue
 			}
 
-			// check that the node is up
-			_, err = client.Health(ctx)
-			if err != nil {
+			if _, err = client.BroadcastTxSync(ctx, tx); err != nil {
 				continue
```

Comment:
Worth logging the failure?

Reply:
we never did before and I think this could be transient, so I think it makes sense to preserve the previous mode.

Reply:
One reason we didn't log it before (I believe) is that `BroadcastTxSync` returns an error when the mempool is full. If we were to log errors and this happens in the testnet it would completely clutter up the logs